src/chrome/content/rules/Riseup.xml

```diff
@@ -1,24 +1,25 @@
 <!--
 	Insecure cookies are set for these domains:
-
 		- labs.riseup.net
 
+	return an HPKP error:
```

Comment:
I get HSTS/mismatch, but there is no HPKP.
See also  george.riseup.net (refused) and  dev.riseup.net (404). Both would work over http.